go.mod

```diff
@@ -14,6 +14,8 @@ require (
 require (
 	github.com/blang/semver v3.5.1+incompatible // indirect
 	github.com/btcsuite/btcutil v1.0.3-0.20201208143702-a53e38424cce // indirect
+	github.com/cespare/reflex v0.3.1 // indirect
```

Comment:
go mod tidy? assuming it got installed with attempt to `go install` at some point